### ISSUE TITLE
Wrong display of `\div` in LaTeX environment which should be with the `physics` package

### ISSUE THREAD

**Issue by Nikucyan** [typora/typora-issues#4715]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [x] Searched http://support.typora.io/

----

**Describe the bug**
After updating the latest version of Typora on Windows, all the divergence symbols in LaTeX mathematic environment (which should look like ) changed to the division symbol. 
The division symbol in LaTeX without any package is `\div`, while with the `physics` package the division symbol is re-defined as `\divsymb`  and `\div` becomes the divergence symbol. (Without the `physics` package, the divergence symbol should be the combination of `\nabla \cdot`)

**To Reproduce**
In LaTeX environment: input `$\div$`

**Expected behavior**
`$\div$` should show a similar result as the combination of `$\nabla \cdot$`

**Screenshots / Screencasts**
The wrong result:
![Wrong](https://cdn.jsdelivr.net/gh/Nikucyan/MD_IMG/img/Snipaste_2021-10-11_01-42-37.png)
The expected result: (in real LaTeX editor w/ `physics` package)
![Right](https://cdn.jsdelivr.net/gh/Nikucyan/MD_IMG/img/Snipaste_2021-10-11_01-46-10.png)

**Sample Markdown File** 
``` markdown
$$
\int_{\text{CV}} \left( \frac{\partial \rho}{\partial t} + \div \rho \vb{v} \right)\ \mathrm{d}V = 0
$$
```

**Desktop (please complete the following information):**
 - OS: Windows 64bit
 
**Typora Version**
0.11.8

**Additional context**
This problem hasn't appeared in the previous versions. I guess it's a result of the wrong lower priority of the `Physics` package.

**Comment by abnerlee** [comment]
added option in preference panel in 0.11.16 to allow redefine from physics package or not.

**Comment by abnerlee** [comment]
#4989

**Comment by shiangsir** [comment]
> > Currently we introduce all packages by default (so user can use it without config)
> > But we do not want `physics` package to override some original TeX commands, so if there are conflicts, we will use origin TeX commands. And `physics` is newly included after 0.11.0, so for compatibility reasons, old one are picked when there are conflicts,
> > I'm actually not a LaTeX user, so is there alternatives for \div in physics package?
> 
> Thanks for the reply. The normal `\div` is the division symbol (÷) and it is redefined to `\divsymb` in `physics`. In `physics`, `\div` means divergence which is the gradient symbol (inverted triangle ▽) with a dot. My screenshots have already shown the 2 different results, I got your point but actually the division symbol has been redefined in the `physics` package. This won't conflict.


Note that the normal `\div` is the division symbol (÷) and it is redefined to `\divisionsymbol` in `physics`.
